CHANGELOG.md

```diff
@@ -80,6 +80,8 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Add utils for dictionary type
 - Filter institute cases using multiple HPO terms
 - Vulture GitHub action to identify and remove unused variables and imports
+- Add a toggle for melter rerun monitoring of cases
```

Comment:
It has been a while, so these 2 should be moved up in the changelog

Reply:
Indeed! I thought I did that in a merge, but time flies.